File: api/src/main/java/br/cefet/tutorParticular/model/Disciplina.java
package br.cefet.tutorParticular.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Disciplina {
    private int idDisciplina;
    private String nome;
    private String descricao;
    private int idAdministrador;
    
}

File: projeto/src/app/model/disciplina.ts
export class Disciplina {
    idDisciplina: number;
    nome: string;
    descricao: string;
    idAdministrador: number;

    constructor(){
        this.idDisciplina = 0;
        this.nome = "";
        this.descricao = "";
        this.idAdministrador = 0;
    }
}
